### PR TITLE
Add missing SetATownInfluenceMultiplier LHScriptX function

### PR DESCRIPTION
**Pull request by raffclar**

### CODE REVIEW

File: src/LHScriptX/FeatureScriptCommands.cpp [hunk 1]
```diff
@@ -201,6 +202,11 @@ inline glm::vec3 GetSize(int size)
 	return glm::vec3(size, size, size) * 0.001f;
 }
 
+void FeatureScriptCommands::SetATownInfluenceMultiplier(int32_t townId, float mult)
+{
+	SPDLOG_LOGGER_ERROR(spdlog::get("scripting"), "LHScriptX Function {} not implemented.", __func__);
```
Comment (bwrsandman) [inline src/LHScriptX/FeatureScriptCommands.cpp:207]: The following will kill the warnings and make it easier to debug/implement in the future.
```c+
SPDLOG_LOGGER_ERROR(spdlog::get("scripting"), "LHScriptX Function {}(townId={}, multiplier={}) not implemented.", __func__, townId, multiplier);

Not gonna block you on this, but I prefer `multiplier` to `mult`
```

**Review by bwrsandman — CHANGES_REQUESTED** [review]

**Comment by bwrsandman** [comment]
Everything else looks great!

**Review by bwrsandman — APPROVED** [review]
Once you've fixed the format error, you can merge.

File: src/LHScriptX/FeatureScriptCommands.cpp (hunk 1 above)
Comment (raffclar) [reply src/LHScriptX/FeatureScriptCommands.cpp]: Done

**Pull request merged**